lib/event-handlers.js

```diff
@@ -1,20 +1,21 @@
-var props = require('../resources/properties');
+'use strict';
+const props = require('../resources/properties');
 
-var registerEventHandlers = function (eventHandlers) {
-  eventHandlers.onSessionStarted = function (sessionStartedRequest, session) {
-    console.log('MetroTransit onSessionStarted requestId: ' + sessionStartedRequest.requestId +
-        ', sessionId: ' + session.sessionId);
+const registerEventHandlers = (eventHandlers) => {
```

Comment:
I don't know how I feel about this one, because I see the const not affecting the scope of the function declaration. I actually prefer to have functions defined as `functionName(functionArguments) {}`.